Dockerfile

```diff
@@ -30,6 +30,7 @@ COPY webhooks/ webhooks/
 
 COPY hack/boilerplate.go.txt hack/boilerplate.go.txt
 COPY hack/csv-generator.go hack/csv-generator.go
+COPY hack/prom_metric_linter.sh hack/prom_metric_linter.sh
```

Comment:
Note: I can't add comment to the relevant line, but I mean to line #4 in the Dockerfile.

Can you add `podman` here:
```
RUN microdnf install -y make tar gzip which podman && microdnf clean all
```
In the dockerfile? it should be added during build stage, since `make manager` is executed during that stage (`manager` is executing `lint`).

Reply:
done

Reply:
@codingben This doesn't seem to work. Any other ideas?

Reply:
When running `docker build .` locally, I get:
```
19.62 ./hack/prom_metric_linter.sh --operator-name="kubevirt" --sub-operator-name="ssp"
67.21 cannot clone: Operation not permitted
67.21 Error: cannot re-exec process
67.21 no working container runtime found. Podman seems not to work.
67.21 make: *** [Makefile:329: lint-metrics] Error 1
------
ERROR: failed to solve: failed to solve with frontend dockerfile.v0: failed to build LLB: executor failed running [/bin/sh -c CGO_ENABLED=0 GOOS=linux GOARCH=amd64 GO111MODULE=on make manager]: runc did not terminate sucessfully
```

Reply:
I guess so, isn't it possible? How can we configure it?

Reply:
I don't think it is possible. I suggest making the metrics linter target a dependency of the `unittest` target. This way the metrics linter runs in PRs but not in the build.